src/firefly/js/templates/lightcurve/LcViewer.jsx

```diff
@@ -216,9 +216,30 @@ UploadPanel.defaultProps = {
     name: 'LCUpload'
 };
 
+let plotConfig = {
+    "plot": {
+        "periodogram": {
+            "x": {
+                "name": "Period"
+            },
+            "y": {
+                "name": "Power"
+            }
+        },
+        "peaks": {
+            "x": {
+                "name": "Power"
+            },
+            "y": {
+                "name": "Peak"
+            }
+        }
+    }
+};
```

Comment:
Is funny, for me is rather easier to read the other way but is probably less lines to read so yes i can clean it up.